bottomsheet/src/main/java/com/flipboard/bottomsheet/BottomSheetLayout.java

```diff
@@ -517,6 +528,10 @@ public void onAnimationEnd(@NonNull Animator animation) {
      * Set the presented sheet to be in a peeked state.
      */
     public void peekSheet() {
+        if (getSheetView().getMeasuredHeight() < BottomSheetLayout.this.getMeasuredHeight()) {
```

Comment:
Ah wait, I see where you mentioned this now. The logic seems backward though, shouldn't it only expand if the measured height is equal to or greater than the layout?

Also, I think this would only apply in cases of wrap_content, which should be checked as well